controllers/integration_test.go

```diff
@@ -1150,7 +1248,7 @@ var _ = Context("INTEGRATION: Inside of a new namespace", func() {
 
 			// Scale-up to 2 replicas on first workflow_job webhook event
 			{
-				env.SendWorkflowJobEvent("test", "valid", "pending", "created", []string{"self-hosted"})
+				env.SendWorkflowJobEvent("test", "valid", "pending")
```

Comment:
We need to simulate the default and custom labels to make these tests passing. I'll follow-up with a commit to fix that.